connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -150,10 +156,30 @@ private void loadInitialConsumerGroups()
 
     List<String> findConsumerGroups()
             throws InterruptedException, ExecutionException {
-        return listConsumerGroups().stream()
+        List<String> filterGroups = listConsumerGroups().stream()
```

Comment:
Nit: "filtered" instead of "filter"

```suggestion
        List<String> filteredGroups = listConsumerGroups().stream()
```

Reply:
nice!